sematic/cli/version.py

```diff
@@ -5,15 +5,27 @@
 import click
 
 # Sematic
+from sematic import api_client
 from sematic.cli.cli import cli
-from sematic.versions import CURRENT_VERSION_STR, MIN_CLIENT_SERVER_SUPPORTS_STR
+from sematic.config.config import switch_env
+from sematic.versions import CURRENT_VERSION_STR, version_as_string
 
 
 @cli.command("version", short_help="Print version information.")
 def version():
     """
     Print the Server, Client, and Python versions.
     """
-    click.echo(f"Sematic server v{CURRENT_VERSION_STR} is installed.")
-    click.echo(f"Client versions >= v{MIN_CLIENT_SERVER_SUPPORTS_STR} are supported.")
-    click.echo(f"Python v{platform.python_version()} is running this binary.")
+    switch_env("user")
```

Comment:
So we get the actual server the user's code will talk to, and not always the local server.